bootstrap.d/net-dns.yml

```diff
@@ -0,0 +1,72 @@
+packages:
+  - name: libidn2
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: An implementation of the IDNA2008 specifications (RFCs 5890, 5891, 5892, 5893)
+      description: This package provides a library for internationalized string handling based on standards from the Internet Engineering Task Force (IETF)'s IDN working group, designed for internationalized domain names.
+      spdx: 'LGPL-3.0-or-later'
+      website: 'https://www.gnu.org/software/libidn/#libidn2'
+      maintainer: "Dennis Bonke <dennis@managarm.org>"
+      categories: ['net-dns']
+    source:
+      subdir: 'ports'
+      git: 'https://gitlab.com/libidn/libidn2.git'
+      tag: 'v2.3.2'
+      version: '2.3.2'
+      tools_required:
+        - host-autoconf-v2.69
+        - host-automake-v1.15
+        - host-libtool
+        - host-pkg-config
+      regenerate:
+        - args: ['./bootstrap']
+        - args: ['autoreconf', '-fvi']
+    tools_required:
+      - system-gcc
+      - host-autoconf-v2.69
+      - host-automake-v1.15
+      - host-libtool
+      - host-pkg-config
+    pkgs_required:
+      - mlibc
+      - libunistring
+      - libiconv
+    configure:
+      # Remove some files from the source directory if they exist.
+      - args: |
+              if [ -f @THIS_SOURCE_DIR@/lib/gendata ]; then
+                  rm -rv @THIS_SOURCE_DIR@/lib/gendata
+              fi
+      - args: |
+              if [ -f @THIS_SOURCE_DIR@/lib/gentr46map ]; then
+                  rm -rv @THIS_SOURCE_DIR@/lib/gentr46map
+              fi
```

Comment:
Why is this done?

Reply:
In a clean source, these shouldn't exist. If they do exist, they can cause issues on rebuilds.